pkg/common/x509util/serialnumber.go

```diff
@@ -3,26 +3,29 @@ package x509util
 import (
 	"crypto/rand"
 	"fmt"
-	"math"
 	"math/big"
 )
 
 var (
-	maxUint64 = new(big.Int).SetUint64(math.MaxUint64)
-	one       = big.NewInt(1)
+	maxUint128, ok = new(big.Int).SetString("340282366920938463463374607431768211455", 10) //(2^128 − 1)
```

Comment:
albeit unlikely such a bug would make it out to pruduction, we should probably panic early if this is fails to parse:

```
maxUint128 = mustSetString("340282366920938463463374607431768211455", 10) //(2^128 − 1)
```